include/vinecopulib/bicop/elliptical.hpp

```diff
@@ -9,6 +9,11 @@
 #include <vinecopulib/bicop/parametric.hpp>
 
 namespace vinecopulib {
+
+namespace constant {
+    constexpr double pi = 3.14159265358979323846;
```

Comment:
Is there a good reason for removing the ifndef? The M_PI is a standard I think...

Reply:
yeah from C ;) constexpr is modern C++

Reply:
My issue is not with the constexpr, it's with using our own definition. How about using stuff from [boost](https://www.boost.org/doc/libs/1_37_0/libs/math/doc/sf_and_dist/html/math_toolkit/toolkit/internals1/constants.html) ?

Reply:
Using our own def is just more leightweight, but we can use boost if you prefer (I will add more digits in the meanwhile).